### DISCUSSION TITLE
workflow alias in same nf file?

### DISCUSSION THREAD

**Discussion by wangzc66** [nextflow-io/nextflow#4930] [Q&A] [answered]
Hi,

How to define aliases for a workflow in the same nf file?
I have tried using `include` before， but It doesn't work.
```
workflow w {
  ...  
}


workflow  {

  w1()
  w2()
  w3()
  
}
```
Thanks.

**Comment by bentsherman** [comment, marked as answer]
Don't think it's possible to alias within a script, you have to declare the workflow `w` in a separate script and include it as a module in order to do the aliasing
